cassiopeia/solver/graph_utilities.py

```diff
@@ -43,14 +43,14 @@ def construct_connectivity_graph(
     with distant mutations.
 
     Args:
-        cm: The character matrix of observed character states for all samples
+        character_matrix: The character matrix of observed character states for all samples
```

Comment:
Noted above -- if this is a user-facing function, shall we make it a bit more usable and pass in an actual character matrix, with string names, that we convert to a numpy array within the function?